cmd/palomar/README.md

```diff
@@ -1,37 +1,33 @@
-# palomo
+# Palomar
 
-An elasticsearch/opensearch frontend and ATP repo crawler meant to provide
-search services for the bluesky network.
+Palomar is an Elasticsearch/OpenSearch frontend and ATP (ActivityPub) repository crawler designed to provide search services for the Bluesky network.
```

Comment:
looking at this now, we are prefering `atproto` over `ATP` in this kind of context. but more importantly, this isn't "ActivityPub", it is "AT Protocol".

Reply:
Understood, my mistake.